frontend/src/queries/utils.ts

```diff
@@ -217,13 +217,13 @@ export function taxonomicFilterToHogQl(
     value: TaxonomicFilterValue
 ): string | null {
     if (groupType === TaxonomicFilterGroupType.EventProperties) {
-        return `properties.${value}`
+        return `properties."${value}"`
     }
     if (groupType === TaxonomicFilterGroupType.PersonProperties) {
-        return `person.properties.${value}`
+        return `person.properties."${value}"`
     }
     if (groupType === TaxonomicFilterGroupType.EventFeatureFlags) {
-        return `properties.${value}`
+        return `properties."${value}"`
     }
```

Comment:
Should we support escaping and unescaping as well? This will come back to bite a support hero one day ("mushroom debt"?). See [hogql/escape_sql.py](https://github.com/PostHog/posthog/blob/c18931e7ff7482d7f6b99518e068a094a6805801/posthog/hogql/escape_sql.py) for what Python does. Ask ChatGPT to translate that to our common tongue, ~~English~~ JavaScript

Reply:
What do you mean by "escaping and unescaping" as an example of a problem here?

Reply:
This is a rare, yet legit edge case for a property name:

```
properties.`my "happy" property with a random \` backquote`
```

It's fine if we "encode" and "decode" it with a simple "add/remove quotes" method, no worries. However we might dump this string as-is into a larger HogQL query somewhere, causing it to just crash. I don't see any danger in shipping this as is, just more support burden somewhere down the line.

Reply:
I see. I'll punt on it though, since it appears only 0.00011% of property definitions on US Cloud contain both types of quotes, and [they all look like someone was just poking us security-wise](http://metabase-prod-us/question#eyJkYXRhc2V0X3F1ZXJ5Ijp7InR5cGUiOiJuYXRpdmUiLCJuYXRpdmUiOnsicXVlcnkiOiJTRUxFQ1QgKiBGUk9NIHBvc3Rob2dfcHJvcGVydHlkZWZpbml0aW9uIFdIRVJFIG5hbWUgTElLRSAnJVwiJScgQU5EIG5hbWUgTElLRSAnJWAlJyIsInRlbXBsYXRlLXRhZ3MiOnt9fSwiZGF0YWJhc2UiOjM0fSwiZGlzcGxheSI6InNjYWxhciIsInBhcmFtZXRlcnMiOltdLCJ2aXN1YWxpemF0aW9uX3NldHRpbmdzIjp7fX0=).